spec/features/main_takewaay_feature_spec.rb

```diff
@@ -0,0 +1,10 @@
+feature 'customer wants to order a meal' do
+  scenario 'customer wants to see a list of dishes and prices' do
+    quiche = Dish.new("food", "price")
```

Comment:
I can see where you're going with this. But, assuming you want this Dish to be a quiche, why not Dish.new("quiche", 3). The second argument being the price.